Breeding-Insight/bi-web
# [BI-1398] - Ontology preview: Checking trait editability status

# Description
**Story:**  [BI-1398 - Ontology preview: Checking trait editability status](https://breedinginsight.atlassian.net/browse/BI-1398)

Added fromImportTable prop to TraitDetailPanel from vue and altered logic and passed props from TraitsImportTable.vue to not have "Checking Trait Editability Status" or an edit option or "Not editable because this trait has associated experiment data." show up when looking at details for an ontology term on import.

# Dependencies
bi-api/develop

# Testing

- Import ontology
- Click show details on one term - no editability status message should appear
- Confirm and go to ontology table
- Click show details - editability behavior should remain as expected for ontology table

# Checklist:

- [ ] I have run TAF: [_\<link to TAF run>_](https://github.com/Breeding-Insight/taf/actions/runs/2332689456)
